### DISCUSSION TITLE
querylog.ignored or statistics.ignored how to use it?

### DISCUSSION THREAD

**Discussion by fate8383** [AdguardTeam/AdGuardHome#5489] [Q&A] [answered]
latest version has this, not in the UI settings as says in the notes but how to do it in the yaml?
querylog.ignored or statistics.ignored

**Comment by ammnt** [comment]
@fate8383, use the instructions from changelog:
https://github.com/AdguardTeam/AdGuardHome/releases

**Reply by fate8383** [reply]
The ability to exclude domain names from the query log or statistics by using the new querylog.ignored or statistics.ignored fields


You mean this?

**Reply by ammnt** [reply]
@fate8383, I mean this section with samples from changelog:
![image](https://user-images.githubusercontent.com/45385632/219301189-6b1f9366-3da6-43bb-8941-91256159d031.png)

**Comment by ainar-g** [comment, marked as answer]
See the [wiki page about the configuration file](https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration):

 >  `ignored`: List of host names, which should not be written to log.
 >
 >  `ignored`: List of host names, which should not be counted.

So, you just put the hostnames that you want to ignore.  For example:

```yaml
# …
querylog:
  # …
  ignored:
  - 'slackb.com'
```

**Reply by fate8383** [reply]
Thanks

**Reply by Quijib0** [reply]
Are there any plans to add this to the UI?  I have isolated access via SSH to my router physically, by port, and making changes to the YAML file is inconvenient.  I would like to be able to blacklist url's from the query log and also be able to delist them at a moments notice from the UI.

**Reply by ainar-g** [reply]
The API and UI are in progress.

**Reply by Quijib0** [reply]
Great news.  Thanks.
